src/proxy/server.rs

```diff
@@ -251,21 +242,13 @@ where
             _p: (),
         };
 
+
```

Comment:
Probably the first valuable review comment I can make with this rust code. Extra whitespace 😂